superset/config.py

```diff
@@ -1923,7 +1923,7 @@ class ExtraDynamicQueryFilters(TypedDict, total=False):
             "Failed to import config for %s=%s", CONFIG_PATH_ENV_VAR, cfg_path
         )
         raise
-elif importlib.util.find_spec("superset_config") and not is_test():
```

Comment:
NOTE: this seemed to say "if you're in test mode, DONT load ANY custom config", which is a confusing hack and probably doesn't belong here or anywhere